### ISSUE TITLE
Update docker-py page on PyPI

### ISSUE THREAD

**Issue by WouldYouKindly** [docker/docker-py#2542]
Hi! Description of https://pypi.org/project/docker-py/ gives no hint that the package is deprecated. Would be lovely to fix that, cause it creates much confusion, and leads to installing both packages which breaks them completely.

One big `DEPRECATED, use https://pypi.org/project/docker/` will suffice, I think :)

**Comment by bailey-dojofive** [comment]
Yeah... just spent more time than I'd like to admit not noticing I'd installed using the name of the repo instead of just `docker` and since that worked happily, with no warning or deprecation message... wondering why `client.pull()` worked but not `client.images.pull()`.
